optaplanner-core/src/main/java/org/optaplanner/core/config/solver/SolverConfig.java

```diff
@@ -82,7 +84,8 @@
         "domainAccessType",
         "scoreDirectorFactoryConfig",
         "terminationConfig",
-        "phaseConfigList"
+        "phaseConfigList",
+        "metricConfig",
```

Comment:
Let's not expose it in the solver config just yet. Let's give it a while to settle down.